docs/site/content/docs/assets/prereq-unmanaged-linux.md

```diff
@@ -0,0 +1,3 @@
+| Architecture   | CPU | RAM  | Required software |
+|:---------------|:----|:-----|:------------------|
+| x86_64 / amd64 | 1   | 2 GB | [Docker engine](https://docs.docker.com/engine/install) |
```

Comment:
It can probably deploy with 1 GB, but do we really want to state that as a minimum?

Reply:
Do you mean 1 CPU? Minimum is `2 GB`.

Reply:
Yeah, distracted. I meant 1 CPU.

Reply:
I think 1 CPU may be adequate. Do you have a machine (non-linux/docker desktop) that you could validate with?

Reply:
I could spin up a linux host, but otherwise no, I don't. But even linux it seems like it would be safer to have more than 1 CPU if they want to actually run anything on the cluster.

But let's leave this at 1 for now until we hear feedback that would make us want to change it.

Reply:
i'll do a test with qemu

Reply:
@stmcginnis quick profiling of unmanaged-cluster running with default config:

![profiling-unmanaged](https://user-images.githubusercontent.com/6200057/149062148-e70cad78-d947-4125-922f-620666d54028.gif)

Since CPU is a compressible/scheduled resource, it had no issues during bootstrapping too (even though it hit 100 a few times)